.github/workflows/package-command.yaml

```diff
@@ -18,13 +18,13 @@ on:
     types: [package-command, Package-command, PACKAGE-command]
 jobs:
   solutionNameDetails:
-    if: ${{ !github.event.pull_request.head.repo.fork }}
+    if: ${{ github.actor != 'dependabot[bot]' && !github.event.pull_request.head.repo.fork }}
     uses: ./.github/workflows/getSolutionName.yaml
     secrets: inherit
 
   # BELOW JOB WILL CHECK IF WE NEED TO SKIP PACKAGE CREATION OR NOT
   checkSkipPackagingDetails:
-    if: ${{ needs.solutionNameDetails.outputs.solutionName != '' && !github.event.pull_request.head.repo.fork }}
+    if: ${{ github.actor != 'dependabot[bot]' && needs.solutionNameDetails.outputs.solutionName != '' && !github.event.pull_request.head.repo.fork }}
```

Comment:
Please enclose the static value in const or at global env variable

Reply:
Creating a common environment variable and using it in `if` condition is not allowed. This is based on official github doc as a reference 

![image](https://github.com/Azure/Azure-Sentinel/assets/107389644/60f06df7-cbf9-4c39-823e-73d7ee1c7aba)
![image](https://github.com/Azure/Azure-Sentinel/assets/107389644/6de62090-50b8-4b00-bd32-a4ff0c1751db)